CHANGELOG.md

```diff
@@ -4,6 +4,48 @@ Releases
 This page documents the releases of Sleeper. Performance figures for each release
 are available [here](docs/12-performance-test.md)
 
+## Version 0.18.0
+This contains the following improvements:
+
+Bulk Import:
+- Support use of EMR serverless.
+- Support use of Graviton instances in bulk import EMR.
+- Report on validation status for bulk import jobs.
+- Added option to query rejected jobs in the `IngestJobStatusReport`.
+- Support use of instance fleets in bulk import EMR.
+- Updated default x86 instance types to use m6i equivalents.
+
+Environment:
+- Allow multiple users to access cdk-environment EC2.
+- Allow users to connect to an existing deployed environment without redeploying.
+
+Deployment:
+- Added ability to deploy to multiple subnets.
+- Split properties templating from the deployment process.
+- Added retry and wait for running ECS tasks when capacity is unavailable.
+- Updated performance test documentation.
+
+Tests:
+- Remove irrelevant properties from system test configurations.
+
+Documentation:
+- Added high-level Sleeper design diagram.
+- Added details on how to contribute and sign the CLA [here](CONTRIBUTING.md).
+
+Misc:
+- Added issue and pull request templates.
+- Added VSCode configuration files.
+- Update and manage several dependencies to resolve CVEs found by dependency check.
+
+Bugfixes:
+- Fixed an issue where the file count would not calculate correctly when sending a job with a directory to standard ingest.
+- Fixed an issue where the file count would not calculate correctly when sending a job with a directory to bulk import.
+- Fixed an issue where the file count would not calculate correctly when sending a request with a directory to the ingest batcher.
```

Comment:
These could be one line, and could be specific that files under a directory were not counted in the reports.